monkey/monkey_island/cc/app.py

```diff
@@ -170,10 +177,12 @@ def init_api_resources(api: FlaskDIWrapper):
     api.add_resource(PropagationCredentials, "/api/propagation-credentials/<string:guid>")
     api.add_resource(RemoteRun, "/api/remote-monkey")
     api.add_resource(VersionUpdate, "/api/version-update")
```

Comment:
Rename to just version or version-info? This doesn't update anything

Reply:
Just `version` should be sufficient.